services/storage/tests/unit/test_async_jobs.py

```diff
@@ -395,7 +229,7 @@ async def test_get_data_export_status(
     ],
     indirect=["mock_celery_client"],
 )
-async def test_get_data_export_status_error(
+async def test_async_jobs_status_error(
```

Comment:
this guy is full of mock client stuff. this has already shown to be something that fail. Please remove these or look with @giancarloromeo or @bisgaard-itis on how to remove them.
Otherwise we check on Tuesday.

Reply:
I've removed all the mocks here.